cve_bin_tool/format_checkers.py

```diff
@@ -76,9 +78,55 @@ def update_checker_table(file_path, markdown):
         f.writelines(lines)
 
 
+def insort_case_insensitive(list, x):
+    key = x.lower()
+    lo, hi = 0, len(list)
+    while lo < hi:
+        mid = (lo + hi) // 2
+        if key < list[mid].lower():
+            hi = mid
+        else:
+            lo = mid + 1
+    list.insert(lo, x)
+
+
+def update_allowed_words(checkers_array):
+    """Updates the allowed_words.py file with the new checkers"""
+    checkers_words = re.findall(r"[^0-9_]+", "_".join(checkers_array))
+    fileObj = open(
+        os.path.join(
+            os.path.abspath("."), ".github", "actions", "spelling", "allow.txt"
+        ),
```

Comment:
You can set the path to a constant or variable to avoid duplication here and in line 116-118.